src/Telemetry/AzureDevOpsTimeline/AzureDevOpsTimeline.cs

```diff
@@ -231,6 +237,45 @@ private async Task RunLoop(CancellationToken cancellationToken)
                         issue.AugmentedIndex = "999." + issue.Index.ToString("D3");
                     }
                 }
+            }      
+
+            TimeSpan cancelationTime; 
+            if(!TimeSpan.TryParseExact(_options.Value.LogScrapingTimeout, @"m\:s\:fff", CultureInfo.CurrentCulture, out cancelationTime)){
```

Comment:
```suggestion
            if(!TimeSpan.TryParseExact(_options.Value.LogScrapingTimeout, @"m\:s\:fff", CultureInfo.CurrentCulture, out cancelationTime))
            {
```

Reply:
Plus, suggest adding Trace log message that is not able to parse `LogScrapingTimeout`; cause, imagine someone update format of `TimeSpan.TryParseExact` and the code will always set `cancelationTime` to 10 mins.

Reply:
We changed it to just `TimeSpan.Parse`, so the app won't deploy successfully if this is the case, so there's no need to check this now